js/common/modules/jsunity.js

```diff
@@ -133,6 +143,15 @@ function Run (testsuite) {
       console.error('unknown function: %s', key);
     }
   }
+  if (tests.length === 0) {
+    print(`There is no test in testsuite "${suite.suiteName}" or your filter "${testFilter}" didn't match on anything`);
+    return {
+      duration: 0,
+      status: true,
```

Comment:
Does this mean that no error is returned?
So when filtering on non-existing tests (and no tests are executed) this counts as a "success"?